index.html

```diff
@@ -0,0 +1,83 @@
+                    Elvin
+                    Charles Stakman, he returns to school study under Stakman, who teaches him about breeding
+                    pest-resistent
+                    plants.</li>
+                <li><b>1941</b> - Tries to enroll in the military after the Pearl Harbor attack, but is rejected.
+                    Instead, the
+                    military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other
+                    applied
+                    science.</li>
+                <li> <b>1942</b> - Receives a Ph.D. in Genetics and Plant Pathology</li>
+                <li><b>1944</b> - Rejects a 100% salary increase from Dupont, leaves behind his pregnant wife, and flies
+                    to
+                    Mexico to head a new plant pathology program. Over the next 16 years, his team breeds 6,000
+                    different strains
+                    of disease resistent wheat - including different varieties for each major climate on Earth.</li>
+                <li> <b>1945</b> - Discovers a way to grown wheat twice each season, doubling wheat yields</li>
+                <li><b>1953</b> - crosses a short, sturdy dwarf breed of wheat with a high-yeidling American breed,
+                    creating a
+                    strain that responds well to fertilizer. It goes on to provide 95% of Mexico's wheat.</li>
+                <li><b>1962</b> - Visits Delhi and brings his high-yielding strains of wheat to the Indian subcontinent
+                    in time
+                    to help mitigate mass starvation due to a rapidly expanding population</li>
+                <li> <b>1970</b> - receives the Nobel Peace Prize</li>
+                <li> <b>1983</b> - helps seven African countries dramatically increase their maize and sorghum yields
+                </li>
+                <li> <b>1984</b> - becomes a distinguished professor at Texas A&M University</li>
+                <li><b>2005</b> - states "we will have to double the world food supply by 2050." Argues that genetically
+                    modified crops are the only way we can meet the demand, as we run out of arable land. Says that GM
+                    crops are
+                    not inherently dangerous because "we've been genetically modifying plants and animals for a long
+                    time. Long
+                    before we called it science, people were selecting the best breeds."</li>
+                <li> <b>2009</b> - dies at the age of 95.</li>
+            </ul>
+            <div id="quote">
+                <p>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's
+                    towering
+                    intellect, persistence and scientific vision can make to human peace and progress."</p>
+                <p>-- Indian Prime Minister Manmohan Singh</p>
+            </div>
```

Comment:
- [x] Please use the **q** tag for the quote, since it hols a semantic value
- [x] To reference the _Indian Prime Minister Manmohan Singh_, kindly use the **cite** tag